### PR TITLE
Update template preview dimensions in table layout

### PR DESCRIPTION
**Pull request by jameskoster**
Fixes a regression that snuck in somewhere along the way.

## What?
Reduce the size of the template preview in table layout.

## Why?
It is currently much too large.

## How?
* Add a `width` property to `page-templates-preview-field.is-viewtype-table`
  * Also adds a `max-height` to avoid rows growing too large.
* Adjust the column width so that it hugs its contents.

## Testing Instructions
1. Navigate to the templates data view
2. Enable the preview field
3. Observe the updated dimensions

| Before | After |
| --- | --- |
| Screenshot 2024-07-25 at 14 05 54 | Screenshot 2024-07-25 at 14 04 15 |

### CODE REVIEW

**Comment by jameskoster** [comment]
@youknowriad @ntsekouras @oandregal when you have a moment could I get a review here? 🙏

File: packages/edit-site/src/components/page-templates/style.scss [hunk 1]
```diff
@@ -37,6 +37,8 @@
 	&.is-viewtype-table {
```
Comment (youknowriad) [inline packages/edit-site/src/components/page-templates/style.scss]: Do we still have these classes? I don't think they exist anymore no? you might want to rebase and adapt your code.

**Comment by jameskoster** [comment]
Oh yes, that was changed in another PR.

**Review by youknowriad — APPROVED** [review]
Not entirely sure what to test, but the changes are reasonable.

**Comment by jameskoster** [comment]
In table layout template previews are now smaller than they are on trunk.

**Pull request merged**